openrtb_ext/convert_down.go

```diff
@@ -177,3 +178,122 @@ func moveRewardedFrom26ToPrebidExt(i *ImpWrapper) error {
 
 	return nil
 }
+
+// clear26Fields sets all fields introduced in OpenRTB 2.6 to default values, which
+// will cause them to omitted during json marshal.
+func clear26Fields(r *RequestWrapper) {
+	r.WLangB = nil
+	r.CatTax = 0
+
+	if app := r.App; app != nil {
+		app.CatTax = 0
+		app.KwArray = nil
+
+		if content := r.App.Content; content != nil {
+			content.CatTax = 0
+			content.KwArray = nil
+			content.LangB = ""
+			content.Network = nil
+			content.Channel = nil
+
+			if producer := r.App.Content.Producer; producer != nil {
+				producer.CatTax = 0
+			}
+		}
+
+		if publisher := r.App.Publisher; publisher != nil {
+			publisher.CatTax = 0
+		}
+	}
+
+	if site := r.Site; site != nil {
+		site.CatTax = 0
+		site.KwArray = nil
+
+		if content := r.Site.Content; content != nil {
+			content.CatTax = 0
+			content.KwArray = nil
+			content.LangB = ""
+			content.Network = nil
+			content.Channel = nil
+
+			if producer := r.Site.Content.Producer; producer != nil {
+				producer.CatTax = 0
+			}
+		}
+
+		if publisher := r.Site.Publisher; publisher != nil {
+			publisher.CatTax = 0
+		}
+	}
+
+	if device := r.Device; device != nil {
+		device.UA = ""
+		device.SUA = nil
+		device.LangB = ""
+	}
+
+	if regs := r.Regs; regs != nil {
+		regs.GDPR = nil
+		regs.USPrivacy = ""
+	}
+
+	if source := r.Source; source != nil {
+		source.SChain = nil
+	}
+
+	if user := r.User; user != nil {
+		user.KwArray = nil
+		user.Consent = ""
+		user.EIDs = nil
+	}
+
+	for _, imp := range r.GetImp() {
+		imp.Rwdd = 0
+		imp.SSAI = 0
+
+		if audio := imp.Audio; audio != nil {
+			audio.PodDur = 0
+			audio.RqdDurs = nil
+			audio.PodID = 0
+			audio.PodSeq = 0
+			audio.SlotInPod = 0
+			audio.MinCPMPerSec = 0
+		}
+
+		if video := imp.Video; video != nil {
+			video.StartDelay = nil
```

Comment:
`r.imp[].video.startdelay` is a recommended OpenRTB 2.5 field

Reply:
Oops. Thanks for verifying the fields.